testing/python/metafunc.py

```diff
@@ -286,9 +286,8 @@ def test_unicode_idval(self) -> None:
             ("ação", "a\\xe7\\xe3o"),
             ("josé@blah.com", "jos\\xe9@blah.com"),
             (
-                "δοκ.ιμή@παράδειγμα.δοκιμή",
-                "\\u03b4\\u03bf\\u03ba.\\u03b9\\u03bc\\u03ae@\\u03c0\\u03b1\\u03c1\\u03ac\\u03b4\\u03b5\\u03b9\\u03b3"
-                "\\u03bc\\u03b1.\\u03b4\\u03bf\\u03ba\\u03b9\\u03bc\\u03ae",
+                "δοκ.ιμή@δοκιμή",
```

Comment:
converted bytes now exceed the 100 limit, unsure of the data set so shortened it, not sure if acceptable

Reply:
this one makes me wonder if a "character" limit of 100 might be better than a bytes limit of 100

as far as i can guess the source data is a most interesting mail address and the shortened one isnt

Reply:
I'm curious why we even escape ID names to ASCII? Is there a reason not to allow Unicode IDs?

OK found the answer: `disable_test_id_escaping_and_forfeit_all_rights_to_community_support` hehe.